waspc/examples/todoApp/ext/Main.css

```diff
@@ -35,7 +35,7 @@ button.selected {
     background-color: white;
     border: none;
     display: inline-block;
-    font-size: 24px;
+    font-size: 24px !important;
```

Comment:
This is kind of a hack for now, we should instead figure out a better way to allow user to modify font instead of doing this, but I decided to leave that for later.

Reply:
maybe add comment about this?

Reply:
I would not for now -> when you see !important you already know it is a hack :D!